Sources/MongoSwift/MongoCollection+Write.swift

```diff
@@ -59,7 +59,6 @@ extension MongoCollection {
     ) -> EventLoopFuture<InsertManyResult?> {
         return self.bulkWrite(values.map { .insertOne($0) }, options: options, session: session)
             .flatMapThrowing { InsertManyResult(from: $0) }
-            .flatMapErrorThrowing { throw convertBulkWriteError($0) }
```

Comment:
a newly enabled test in `MongoCollectionTests` caught this. since `insertMany` throws a bulk write error we don't need to do the conversion.